cmd/internal/planetscale_edge_database.go

```diff
@@ -37,6 +37,8 @@ type PlanetScaleEdgeDatabase struct {
 	Logger   AirbyteLogger
```

Comment:
For my own clarification, what will `Logger` be responsible for vs `dataLogger`?